gtsfm/multi_view_optimizer.py

```diff
@@ -112,6 +120,21 @@ def create_computation_graph(
                 two_view_reports_dict,
                 debug_output_dir,
             )
+            (
+                viewgraph_i2Ri1_graph,
+                viewgraph_i2Ui1_graph,
+                viewgraph_v_corr_idxs_graph,
+                viewgraph_two_view_reports_graph,
+                viewgraph_estimation_metrics,
```

Comment:
IMHO this will be overly verbose. We already get visualizations saved for each of the two stages, currently, and I think the combined performance is what matters most.